python/lsst/ts/imsim/closedLoopTask.py

```diff
@@ -575,32 +575,21 @@ def _run_sim(
             sensor_names = np.array(
                 [sensor_wfe.sensor_name for sensor_wfe in list_of_wf_err]
             )
-            field_idx = np.array(
-                [
-                    self.ofc_calc.ofc_data.field_idx[sensor_name]
-                    for sensor_name in sensor_names
-                ]
-            )
 
+            # Use the OFC estimates for the wavefront sensors simulated
             fwhm_dict = {x: y for x, y in zip(sensor_id, fwhm)}
             ofc_fwhm = np.array(
                 [fwhm_dict[sensor_wfe.sensor_id] for sensor_wfe in list_of_wf_err]
             )
 
-            if cam_type == CamType.LsstCam:
-                # For the wavefront sensors the sensor ids
-                # are different than the corresponding field row
-                # index in the sensitivity matrix.
-                self.ofc_calc.set_fwhm_data(ofc_fwhm, field_idx)
-            else:
-                self.ofc_calc.set_fwhm_data(fwhm, sensor_id)
+            self.ofc_calc.set_fwhm_data(ofc_fwhm, sensor_names)
```

Comment:
I am wondering, wouldn't this also work by just using fwhm directly? maybe we could then also change `get_list_of_fwhm_sensor_data` to only return the fwhm? This would further remove some lines of code in the closed loop.

Reply:
You're right. This is left over code from the old ts_ofc version.